File: Lab3/Lab3/Model/Result.cs
using System.Text.Json.Serialization;

namespace Lab3.Model
{
    class Result
    {
        [JsonPropertyName("message")]
        public string Message { get; set; }

        [JsonPropertyName("account")]
        public Account Account { get; set; }

        [JsonPropertyName("realNumber")]
        public long RealNumber { get; set; }

        public override string ToString()
        {
            return $"[[message]: {Message}; [account]: {Account}; [realNumber]: {RealNumber}]";
        }
    }
}

File: Lab3/Java/Lab3/src/model/Result.java
package model;

import com.google.gson.annotations.SerializedName;

public class Result {
    @SerializedName("message")
    private String message;

    @SerializedName("account")
    private Account account;

    @SerializedName("realNumber")
    private long realNumber;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Account getAccount() {
        return account;
    }

    public void setAccount(Account account) {
        this.account = account;
    }

    public long getRealNumber() {
        return realNumber;
    }

    public void setRealNumber(long realNumber) {
        this.realNumber = realNumber;
    }

    @Override
    public String toString() {
        return String.format("{[message]: %s; [account]: %s; [realNumber]: %s}", message, account, realNumber);
    }
}
